tests/proxysql-admin-testsuite.bats

```diff
@@ -145,6 +145,13 @@ fi
   [ "$status" -eq  0 ]
 }
 
+@test "run proxysql-admin --syncusers --add-query-rule ($WSREP_CLUSTER_NAME)" {
+  mysql_exec "$HOST_IP" "$PORT_3" "create user test_query_rule@'%' identified by 'test';"
+  run sudo PATH=$WORKDIR:$PATH $WORKDIR/proxysql-admin --syncusers --add-query-rule
+  echo "$output" >&2
+  [ "$status" -eq  0 ]
+  [ "${lines[4]}" = "  Added query rule for user: test_query_rule" ]
```

Comment:
Test should also verify that the query rules were added to proxysql.

Reply:
Added  test to verify the query rules are present in ProxySQL DB 4ef41369f37bd111a34a5b6ebc8df25cbab7ea21